client/app/js/snackbar.jsx

```diff
@@ -0,0 +1,77 @@
+/*jshint esversion: 6 */
+import Snackbar from 'material-ui/Snackbar';
+import React from 'react';
+
+class MySnackbar extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      open: false,
+      message: "",
+      ws: undefined,
+      boardName: this.props.boardName,
+    };
+  }
+
+  handleTouchTap = () => {
+    this.setState({
+      open: true,
+      message: "Board Updated",
+    });
+  };
+
+  handleRequestClose = () => {
+    this.setState({
+      open: false,
+    });
+    this.boardReloadHandler();
+  };
+
+  boardReloadHandler = () => {
+    this.props.boardReloadHandler();
+  };
+
+  update_websocket = (board_name) => {
+    if (board_name === "LOADING...") {
```

Comment:
This is not a good method to check if the board is still loading. We should not rely on some String in the front-end